spec/lib/tasks/import_spec.rb

```diff
@@ -28,12 +28,14 @@
       Rake::Task["import:whitehall_document"].reenable
       allow(ResyncService).to receive(:call)
       allow(WhitehallImporter::ClearLinksetLinks).to receive(:call)
+      allow(WhitehallImporter::Import).to receive(:call).and_return(create(:document, :with_current_edition))
```

Comment:
I think we've got an issue here where this is also testing a level above the unit of code. 

I've put together a quick proposal for how we should unit test this task: https://github.com/alphagov/content-publisher/commit/a99eed212c58ac7adf7e72727934dbdffe3cfa7e

Reply:
Oh! You want to remove the "creates a document" test? Well that certainly makes the rake task easier to test. I assume it's because the functionality and whether a document is created should be tested by the `WhitehallImporter` tests?

Reply:
Yeah we don't actually create a document in the rake task so it seems an odd thing to assert on, I guess we either used to do that or we were testing this more like an integration test.